modules/ROOT/pages/packstream/index.adoc

```diff
@@ -411,6 +411,7 @@ Unlike other values, there is no separate encoding for byte arrays containing fe
 
 One of the markers `CC`, `CD`, or `CE` should be used, depending on scale.
 This marker is followed by the size and bytes themselves.
+Even though the unsigned 32-bit integer following `CE` could hold a bigger number, the maximum size of byte arrays is limited to 2 147 483 647 (maximum value of a signed 32-bit integer).
```

Comment:
I'd probably swap `even though` for `N.B.` also should these lines render on new lines? the ascii doc on github renders as one paragraph

Reply:
I wasn't sure if I wanted it to be a separate paragraph. A paragraph consisting on one sentence felt a little odd...